### PR TITLE
values update q2 2020

### PR DESCRIPTION
**Pull request by jthandy**
These are long overdue; this is primarily from work I did around values updates for the retreat last year. Please let me know if you have any comments on the wording (or, I suppose, intent if applicable)...I spent a lot of time trying to get this right but your eyes are very much appreciated.

### CODE REVIEW

File: values.md [hunk 1]
```diff
@@ -3,33 +3,50 @@
...
-- We believe that compensation shouldn't create "handcuffs".
+While we pursue the mission with intense focus, we recognize that creating the change in the world that we wish to see will not happen overnight. We think, plan, and act for the long-term.
+
+### We work hard and go home.
+
+We take our responsibilities as parents, citizens, individuals, family members, and members of our various communities just as seriously as we take our commitments at work. When we are on, we are focused. When we are off, we are free to be fully disconnected.
+
+### We treat employees like people.
+
+The only way to build a team of humans who innovate in shared service to a mission is to _treat them like people_. And people only want a few things: economic security, self-determination, health, belonging, and meaning. We structure our compensation and benefits, work hours and location policies, work and management styles, and employee agreements to support this fundamental human-ness.
```
Comment (drewbanin) [inline values.md:43]: I don't love this section -- the verbiage here is too euphemistic and hand-wavy, while also being overly prescriptive about what people _want_. Here's my take, happy to workshop or discuss!

### We are human.

We are human beings innovating in a shared service to a mission. We structure our compensation and benefits, work hours and location policies, work and management styles, and employee agreements to support this fundamental human-ness. We bring our whole selves to work, and we recognize that our identities extend beyond the work that we do.

File: values.md [hunk 2]
```diff
@@ -3,33 +3,50 @@
 We are a mission-driven company. We believe that profits are an important tool in the pursuit of our mission, but they are not the mission itself.
 
 ### We contribute to the knowledge loop.
+
 The highest goal of any human is to produce new knowledge that can subsequently be built upon by others. This is the process upon which every good thing in human society has been built. We participate in this most fundamental of human endeavors by thinking in public and defaulting to open source.
 
 ### We are more concerned with value creation than value capture.
 
-### Work done well conveys dignity.
+We will only succeed if we create value for our community. If we optimize for that first, commercial success will follow.
```
Comment (drewbanin) [inline values.md:11]: I think this is more accurate and more general:

```suggestion
We will only succeed if we create value in the world. If we optimize for that first, commercial success will follow.
```

File: values.md (hunk 1 above)
Comment (jthandy) [reply values.md:43]: this is a fantastic change. I struggled writing this section; yours is much better.

**Comment by jthandy** [comment]
@drewbanin Thanks a lot for your thoughtful review. I've incorporated your changes and am merging!

**Comment by jthandy** [comment]
I failed in this initial PR to provide context around these changes! This was very much my bad, and so I'm adding additional context here; I'll be linking to this in a Slack message to the team shortly. 

First, this PR addresses https://github.com/fishtown-analytics/corp/issues/37. It doesn't do so precisely; in the process there was some editing. The main edit is that this:

> We are patient, yet urgent.

...got transformed into this:

> We optimize for the long-term.

The prior formulation focuses on the tension between maintaining a sense of near-term urgency and a commitment to the long-term. This tension is interesting (and very much still true and relevant), but I felt it was important to keep the statement of value simple: we care about success over the long-term. Simple values are easier to communicate, remember, and implement. This does not detract from the truth of the earlier formulation.

Second, it also reformulates another longstanding value:

> We’re invested in the future of work.

The reformulated version of that reads:

> We treat employees like people.

There was *a lot* of thinking that went into this change. If you're ever interested, take a look at the [history on that particular value](https://github.com/fishtown-analytics/corp/commit/2d3edaca2125ebc264ad51dc07ede3ffa7f7f18b); it's seen a lot of changes since 2016. Originally it was an expression of fairly extreme libertarianism, and a discussion of why exactly I felt that way would require a discussion of:

- why we need stronger social safety nets
- why employers shouldn't provide those, but rather the government should
- why stronger social safety nets provide greater bargaining power on behalf of employees
- why empowered employees with greater bargaining power should be more loosely-attached to firms
- why the firm should, according to Coase, shrink in size (# of employees) as its ability to exchange information with the larger environment increases.

I'm happy to have a conversation about this with you at some point over a beer. Regardless, however, this is not the world we live in today. We needed to update this value to reflect the actual state of the world, where companies shoulder much of the societal responsibility of providing a safety net for their employees. Many firms shirk this responsibility—they pay their employees below a living wage, they don't provide psychologically safe or inclusive work environments, they fail to provide adequate health care and other benefits... Etc. This value is a statement that Fishtown Analytics is made up of people, and we choose to venerate that fact rather than attempt to crush it. 

This value shows up all over the place--from our employment agreements that don't include non-competes, to our benefits that were launched at employee #10 and continue to improve every year, to our always-location-flexible, now-fully-remote stance on work location and extremely flexible stance on work hours. To anyone who has gone through a quarterly review, you also know how human those conversations tend to get.

**Pull request merged**